frontend/catalyst/compiler.py

```diff
@@ -548,6 +390,43 @@ class Compiler:
     def __init__(self, options: Optional[CompileOptions] = None):
         self.options = options if options is not None else CompileOptions()
 
+    @debug_logger
+    def get_cli_command(self, tmp_infile_name, output_ir_name, module_name, workspace):
+        """Prepare the command for catalyst-cli to compile the file.
+
+        Args:
+            module_name (str): Module name to use for naming
+            workspace (Directory): directory that holds output files and/or debug dumps.
+        Returns:
+            cmd (str): The command to be executed.
+        """
+        cli_build_path = get_lib_path("cli", "DIALECTS_BUILD_DIR")
```

Comment:
You will need to document this new variable. Is it not possible to get this location from one of the other variables?

Just a note, this variable refers to an environment variable but I think you may be confusing it with the Makefile variables.